src/AutoCrudAdmin/Helpers/IFormControlsHelper.cs

```diff
@@ -10,7 +10,8 @@ public interface IFormControlsHelper
         IEnumerable<FormControlViewModel> GenerateFormControls<TEntity>(
             TEntity entity,
             EntityAction entityAction,
-            IDictionary<string, Expression<Func<object, bool>>> complexOptionFilters = null);
+            IDictionary<string, Expression<Func<object, bool>>> complexOptionFilters = null,
+            Type autocompleteType = null);
```

Comment:
Add documentation (`//<summary>`) for the method with explanation for this property.

Reply:
Done.

Reply:
This explanation is not very helpful. Docs should reveal more the purpose and usage of something, than what it is by itself. The summary you provided is just the variable name expanded as text.

Reply:
Ok, I gave it a second try.